core/src/test/java/apoc/StartupTest.java

```diff
@@ -39,8 +38,11 @@ public void test() {
 
             neo4jContainer.close();
         } catch (Exception ex) {
-            ex.printStackTrace();
-            fail("Should not have thrown exception when trying to start Neo4j: " + ex);
+            // if Testcontainers wasn't able to retrieve the docker image we ignore the test
+            if (TestContainerUtil.isDockerImageAvailable(ex)) {
```

Comment:
What about using [JUnit assumptions](https://junit.org/junit4/javadoc/4.13/org/junit/Assume.html) instead? This will skip the test entirely.